docs/migration.md

```diff
@@ -360,26 +384,45 @@ Before migrating the volumes make sure the following prerequisites are taken car
+    demo-vol-claim            Bound    pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683   4G         RWO            openebs-jiva-default   29m
+    migrated-demo-vol-claim   Bound    pvc-13001311-8787-4c56-9a26-eef0fa819377   4Gi        RWO            openebs-jiva-csi-sc    10s
+    
+    $ kubectl -n openebs get pvc
+    NAME                                                          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
+    openebs-pvc-13001311-8787-4c56-9a26-eef0fa819377-jiva-rep-0   Bound    pvc-dda3733a-2b7a-49ad-b54b-7e759dee812f   4Gi        RWO            openebs-hostpath   23s
+    openebs-pvc-13001311-8787-4c56-9a26-eef0fa819377-jiva-rep-1   Bound    pvc-1ee9579c-7acf-49f8-b23d-74a2bc0781aa   4Gi        RWO            openebs-hostpath   23s
+
+    $ kubectl -n openebs get pods -o wide
+    NAME                                                              READY   STATUS    RESTARTS   AGE     IP               NODE
+    pvc-13001311-8787-4c56-9a26-eef0fa819377-jiva-rep-0               1/1     Running   0          3m46s   192.168.77.143   kworker2
+    pvc-13001311-8787-4c56-9a26-eef0fa819377-jiva-rep-1               1/1     Running   0          3m46s   192.168.41.148   kworker1
+    ```
+    The node names and paths can be matched by using the above outputs
     ```
-    NodeName 	  Corresponding hostpath 
-    127.0.0.1 	/var/openebs/local/pvc-91d8fa8d-a474-4444-881e-a13571be5487
+    NodeName    Corresponding hostpath 
+    kworker1    /var/openebs/local/pvc-1ee9579c-7acf-49f8-b23d-74a2bc0781aa
+    kworker2    /var/openebs/local/pvc-dda3733a-2b7a-49ad-b54b-7e759dee812f
     ```
 
-6. Scale down the csi replica sts to 0.
+7. Scale down the csi replica sts to 0.
     ```sh
-    $ kubectl scale sts pvc-9cebb2c3-b26e-4372-9e25-d1dc2d26c650-rep -n openebs --replicas=0
+    $ kubectl scale sts pvc-13001311-8787-4c56-9a26-eef0fa819377-jiva-rep -n openebs --replicas=0
     ```
 
-7. SSH into each node. Remove the files from the new localpv hostpath and copy the files from old hostpath to the new localpv hostpath which we have noted down in step 5. The old hostpath would be the same for all replica deployments and will be like `/var/openebs/<pv-name>`, for example `/var/openebs/pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb`
+8. SSH into each node. Remove the files from the new localpv hostpath and copy the files from old hostpath to the new localpv hostpath which we have noted down in step 6. The old hostpath would be the same for all replica deployments and will be like `/var/openebs/<pv-name>`, for example `/var/openebs/pvc-feefe71b-d073-4f0d-a5c2-bc0b78812683`
 
-8. Scale up the replica statefulset. Replace the volume name in the application and scale it up. Verify the data.
+9. Scale up the replica statefulset. Replace the volume name in the application and scale it up. Verify the data.
 
-9. Delete the old volume. Done!!
+10. Delete the old volume. Done!!
     ```sh
```

Comment:
Post this PVC deletion seeing older PV still in released state can we include step to clear this as well